os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -453,7 +490,6 @@ static void recovery_user_assert(void)
 
```

Comment:
For readability and better understanding, Let me suggest another proper name of this function.
static void recovery_user_assert(void) ==> static void quick_recovery_from_user_fault(void)